packages/cdk/lib/constructs/batch.ts

```diff
@@ -124,8 +125,21 @@ export class Batch extends Construct {
         "ebs-autoscaling": new PolicyDocument({
           statements: [
             new PolicyStatement({
-              actions: ["ec2:CreateTags", "ec2:DescribeVolumes", "ec2:CreateVolume", "ec2:AttachVolume", "ec2:DeleteVolume", "ec2:ModifyInstanceAttribute"],
-              resources: ["*"],
+              actions: ["ec2:DescribeVolumes", "ec2:CreateVolume", "ec2:CreateTags"],
+              resources: [`arn:aws:ec2:${Aws.REGION}:${Aws.ACCOUNT_ID}:volume/*`],
```

Comment:
I created a dumb little [util function](https://github.com/aws/amazon-genomics-cli/blob/main/packages/cdk/lib/util/index.ts#L110-L112) for doing something similar with Batch. 

I'm fine with leaving it as-is, but if you think adding a `service` parameter or make a similar function for EC2 ARNs would be more readable then go for it.

Reply:
That's a cleaner approach - I'll add an `ec2Arn` function